test/isLength.test.js

```diff
@@ -0,0 +1,38 @@
+// Importing require so it works on Node.js >14
+import {
+    createRequire
+} from 'module';
+const require = createRequire(
+    import.meta.url);
+    // Chai assert
+    const assert = require('chai').assert;
+    
+    import isLength from "../src/isLength.js";
+    
+    /****** 26.isLength.js tests *****/
+    
+    describe("isLength.js tests", function () {
+        
+        describe("Should return `true` if `value` is a valid length, else `false`", function () {
+            it("should return true if `value` is 3", function () {
+                assert.strictEqual(isLength(3),true);
+            });
+            
+            it("Should return true  if `value` is  Max-safe-integer ", function () {
+                const MAX_SAFE_INTEGER = 9007199254740991;
```

Comment:
Use Number.Max_Safe_integer here

Reply:
Number.Max_Safe_integer is showing error

Reply:
Warning about using it because it is unsafe?